_data/members.yml

```diff
@@ -84,6 +84,16 @@
 - name: Alex Tavasoli
   github: atavasoli
 
+<<<<<<< HEAD
```

Comment:
What's up with the `<<<<<< HEAD` and also at the bottom? Will that appear in the file if merged? (I actually don't know - does someone?)